egs/wsj/s5/run.sh

```diff
@@ -50,7 +61,7 @@ local/wsj_format_data.sh --lang-suffix "_nosp" || exit 1;
  # use local/cstr_wsj_extend_dict.sh --dict-suffix "_nosp" $corpus/wsj1/doc/ instead.
   (
    local/wsj_extend_dict.sh --dict-suffix "_nosp" $wsj1/13-32.1  && \
-   utils/prepare_lang.sh data/local/dict_nosp_larger \
+   utils/prepare_lang.sh $word_class_command_line_option data/local/dict_nosp_larger \
```

Comment:
I chose to call the option/parameter --wclass-opt/$wclass_opt. Hopefully this is better.